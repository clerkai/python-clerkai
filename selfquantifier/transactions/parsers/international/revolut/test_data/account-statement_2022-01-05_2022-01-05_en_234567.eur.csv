Type,Product,Started Date,Completed Date,Description,Amount,Fee,Currency,State,Balance
CARD_PAYMENT,Current,2022-01-03 10:53:10,2022-01-04 11:04:34,apple.com/bill,-9.99,0.00,EUR,COMPLETED,535.96
CARD_PAYMENT,Current,2022-01-05 11:41:21,,Capperi,-33.90,0.00,EUR,PENDING,
