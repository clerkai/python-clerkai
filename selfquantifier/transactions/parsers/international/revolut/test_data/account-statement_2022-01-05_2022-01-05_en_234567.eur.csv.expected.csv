Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
2022-01-03 10:53:10,2022-01-04 11:04:34,,apple.com/bill,-9.99,EUR,535.96,,,,"{'Type': 'CARD_PAYMENT', 'Product': 'Current', 'Started Date': '2022-01-03 10:53:10', 'Completed Date': '2022-01-04 11:04:34', 'Description': 'apple.com/bill', 'Amount': -9.99, 'Fee': 0.0, 'Currency': 'EUR', 'State': 'COMPLETED', 'Balance': 535.96}",2022-01-03 10:53:10,2022-01-04 11:04:34,,apple.com/bill,-9.99,EUR,535.96,,,
